nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -0,0 +1,1240 @@
+"""
+ Copyright (c) 2021 Intel Corporation
```

Comment:
have you modified quantizer_propagation.py content or just splitted into files?
if not, please highlight major changes

Reply:
Better to review commit-wise. Anyway, the common classes have not been changed for the most part, merely moved to common and split into files. The impacted parts are the `TargetPoint` references (which replaced the specific `PTTargetPoint`s)